src/style/list/list.less

```diff
@@ -139,10 +139,21 @@
     cursor: pointer;
   }
 
-  > .left-column-icon {
+  .left-list-item-button {
```

Comment:
I think you can use @kamil-au 's `.button-reset-default` here and just add the `display` and `width` property, can't you?

Reply:
sure, I wasn't aware of this mixin ;)

Reply:
mixin used, thanks